server/budgettracker.business/Services/BudgetService.cs

```diff
@@ -0,0 +1,30 @@
+using System.Threading.Tasks;
+using budgettracker.business.Api.Contracts.BudgetApi;
+using budgettracker.business.Api.Converters;
+using budgettracker.common.Models;
+using budgettracker.data;
+
+namespace budgettracker.business.Services
+{
+    public class BudgetService : IBudgetService
+    {
+
+        private readonly BudgetApiConverter _budgetApiConverter;
+
+        public BudgetService(BudgetApiConverter budgetApiConverter)
+        {
+            _budgetApiConverter = budgetApiConverter;
+        }
+
+        public async Task CreateBudget(BudgetResquestContract budget)
+        {
+            Budget budgetValue = _budgetApiConverter.ToModel(budget);
+
+            using (var budgetContext = new BudgetTrackerContext())
+            {
+                await budgetContext.Budgets.AddAsync(budgetValue);
+                budgetContext.SaveChanges();
```

Comment:
I've been checking for errors here. SaveChanges returns an int of how many items were written to the DB. If that is less than 1, something happened on a DB level so this should notify the user.